### ISSUE TITLE
version 0.5.1 : The "lint" directory does not exist

### ISSUE THREAD

**Issue by rafipiccolo** [VincentLanglet/Twig-CS-Fixer#64]
i update all my tools as part of my ci.

And It used to work well (version 0.5),
But now it errors with latest version (0.5.1) :
```
twig-cs-fixer lint --fix /xxx/views
Error: The "lint" directory does not exist.
```

i reverted version, using this, and it worked again :
```
composer global require vincentlanglet/twig-cs-fixer:0.5
```

has the parameters changed in the new version ? the unit tests passed ?

**Comment by VincentLanglet** [comment]
Hi, can you confirm it's fixed in 0.5.2 ?

**Comment by rafipiccolo** [comment]
yes thx
